test/fixtures_test.js

```diff
@@ -75,6 +75,21 @@ tape.test('verify existance/update fixtures', function(assert) {
             destinations: 'Destination 1,Destination 2'
         });
         checkOrWrite(step, `${basePath}_destination`);
+
+        // exit
+        step = Object.assign(clone(baseStep), {
+            name: 'Way Name',
+            exits: '4A,4B'
+        });
+        checkOrWrite(step, `${basePath}_exit`);
+
+        // exit + destination
+        step = Object.assign(clone(baseStep), {
+            name: 'Way Name',
+            destinations: 'Destination 1,Destination 2',
+            exits: '4A,4B'
+        });
+        checkOrWrite(step, `${basePath}_exit_destination`);
```

Comment:
I went with this approach because `destination` was the most obvious pattern to model this work off of. I also liked how the fixtures guarantee that exit information _isn't_ being added to instructions where it shouldn't be. I'm open to other options, but seems like the additional fixtures are easy enough to maintain.